packages/clay-color-picker/src/__tests__/index.tsx

```diff
@@ -3,8 +3,11 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
+/** @ts-ignore */
```

Comment:
Any specific reason for adding this here?

Reply:
Removed.